lib/grape/endpoint.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'grape/util/header'
```

Comment:
Can we consolidate this require elsewhere once?

Reply:
removed (not required anymore)